### ISSUE TITLE
html output issue when run with only one sample

### ISSUE THREAD

**Issue by stefvanlieshout** [CuppenResearch/bamMetrics#3]
When analysing only one sample, the html output contains this error instead of the table normally present at the end of the document:

```
Error in UseMethod("xtable"): no applicable method for 'xtable' applied to an object of class "character"
```

**Comment by sambrightman** [comment]
The changes in develop branch accidentally fix this by subsetting the transposed summary. However, they break the transposed text and PDF in the single-sample case. #17 fixes both issues (on develop).

**Comment by rernst** [comment]
Confirmed, this is fixed in developed and upcoming release (v2.1.4). 

Thanks @sambrightman for the fix.
